src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -175,6 +176,32 @@ import           Pos.Web.Server                   (serveImpl)
 type WalletWebMode m = m ~ Pos.Wallet.Web.Mode.WalletWebMode
 
 makeLenses ''SyncProgress
+   
+-- | Allows to keep once evaluated `txMempoolToModifier` in context.
+-- Unknown notation? That's ImplicitParams!
+-- If this approach will be going to be used in more places than it currently
+-- is, it may be worth discussing/reconsidering first.
+type WithCachedAccMod = ?cachedAccModifier :: CAccModifier
+
+-- | Evaluates `txMempoolToModifier` and provides result as implicit parameter
+-- to given function.
+fixingCachedAccModifier
+    :: (WalletWebMode m, MonadKeySearch key m)
+    => (WithCachedAccMod => key -> m a)
+    -> key -> m a
+fixingCachedAccModifier action key = do
+    accMod <- txMempoolToModifier =<< findKey key
+    -- @ ? @ doesn't work with @ <- @ :(
+    let ?cachedAccModifier = accMod
+    action key
+
+fixCachedAccModifierFor
+    :: (WalletWebMode m, MonadKeySearch key m)
+    => key
+    -> (WithCachedAccMod => m a)
+    -> m a
+fixCachedAccModifierFor key action =
+    fixingCachedAccModifier (const action) key
```

Comment:
`ImplicitParams` may look cool, but, as I guess, they are unfamiliar to most developers and whoever else will inevitably be confused when reading such code at the first time.

I think that using simple explicit parameter won't make the code worse, instead, the code will be more straightforward and readable, and not much would change, actually - `fixingCachedAccModifier` and `fixCachedAccModifierFor` can be trivially rewritten to explicit parameter, and all functions which now have `WithCachedAccMod` in type can have `CAccModifier ->` instead as well.